lib/resources/r4/related_person_patch.yaml

```diff
@@ -159,3 +159,22 @@ operations:
         <li>Requires a <a href="#test-name-id">test operation</a> to be provided for the `HumanName.id` of the name whose `family` is intended to be replaced.</li>
         <li>This operation is only supported for the first name in the list of related person names (<code>{index}</code> == 0).</li>
       </ul>
+
+  - name: replace-name-suffix
+    path: /name/{index}/suffix
+    operation: replace
+    type: String
+    url: https://hl7.org/fhir/R4/datatypes-definitions.html#HumanName.suffix
+    description: Replace the `HumanName.suffix` of the name at the given <code>{index}</code> in the list of related person names.
+    example: |
+      {
+        "op": "replace",
+        "path": "/name/0/suffix",
+        "value": ["Jr."]
+      }
+    note: >
```

Comment:
let's make a note that only one value for the suffix is supported.

Reply:
added here 194a06e